README.md

```diff
@@ -131,15 +142,26 @@ or
 python scenario_from_csv.py local
 ```
 
+#### Using scenario templates
+It's also possible to adopt the settings of an existing scenario, which we then call a scenario template. To do this, run the following script in the terminal:
+
+```
+python get_template_settings.py
+```
+
+The script will create a `template_settings.csv` file in the `data/output/` folder. This file provides an overview of all slider settings for each scenario template. To adopt the values, open the `data/adopt_template_settings.xlsx` file and copy paste the entire sheet into the `template_settings` sheet. In the `adopter` sheet you can choose which scenario template should be using by specifying the template session ID. Also, you can choose which scenario settings to adopt by specifying TRUE or FALSE for each input key. By default, all settings are adopted except for the ones representing a capacity (in MW). When you're done, replace the `data/input/scenario_settings.csv` by the sheet `scenario_settings` from the Excel file.
```

Comment:
The "adopt_template_settings" is partly in Dutch, such as the sector names and things like "Slider setting overnemen?". Can we use an English version here?